src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs

```diff
@@ -34,6 +34,7 @@ public partial class TestsManifestGeneration
         [ConditionalFact(nameof(IsProcessElevatedAndNotWindowsNanoServerAndRemoteExecutorSupported))]
         public void Test_EventSource_EtwManifestGeneration()
         {
+            RemoteInvokeOptions options = new RemoteInvokeOptions { TimeOut = 600_000 };
```

Comment:
Is this 10 minute timeout being added to address a particular issue #?

Reply:
It was a suggestion when Dan noticed the test failing (https://github.com/dotnet/runtime/pull/58336#issuecomment-909708750). I just used the referenced code, but the timeout could probably be ~5 minutes I'd guess. I'm not familiar with this test, so it's partially an exploratory change.

Reply:
Thanks.  I'm not concerned about the 10 minutes specifically, more that any time I see us raising timeouts to accommodate some sporadic failure in CI, my Spidey sense tingles, either because a) the timeout is being used to address a race condition, b) the timeout is being used to workaround something that takes a long time but could potentially take longer, or c) the original timeout was hit because something deadlocked and no increase in timeout will actually help.  Sounds like this is (b), in which case, the change looks fine, and it'll be interesting to see if this triggers again, in which case a deeper investigation is warranted.

Reply:
Agreed.